src/camera.lua

```diff
@@ -32,7 +33,7 @@ end
 function Camera:setTarget(target)
 	local duration = 1
 	local x, y = self.body:getPosition()
-	local angle = self.body:getAngle()
+	local angle = self.angleFixed and 0 or self.body:getAngle()
```

Comment:
This is the source of the bug: not setting angle to 0 when the camera is in fixed mode. The refactoring gives us access to the fixed mode without needing a reference to the player (which we don't already have in this function).